src/Agent.Worker/TaskManager.cs

```diff
@@ -465,10 +480,8 @@ protected void SetInput(string name, string value)
         }
     }
 
-    public sealed class NodeHandlerData : HandlerData
+    public abstract class BaseNodeHandlerData : HandlerData
```

Comment:
`NodeHandlerDataBase` instead of `BaseNodeHandlerData`. :D

Reply:
`NodeHandlerDataBase` was actually my first option to name the class. That said, the "DataBase" part is kind of confusing (because this is not related to databases) 😄. There appears to not be a naming convention for C# base classes, so I went with avoiding the confusion by prefixing Base instead.